weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/SchemaMappingStep.tsx

```diff
@@ -0,0 +1,352 @@
+import {Box, Paper, Stack, Tooltip, Typography} from '@mui/material';
+import React, {useEffect, useState} from 'react';
+
+import {parseRef} from '../../../../../react';
+import {Select} from '../../../../Form/Select';
```

Comment:
Related to the above comment. GO here and set the page limit to 100: https://app.wandb.test/timssweeney/leaderboard-demo/weave/calls?filter=%7B%22opVersionRefs%22%3A%5B%22weave%3A%2F%2F%2Ftimssweeney%2Fleaderboard-demo%2Fop%2Fgpt_4o_mini_with_context%3A*%22%5D%7D&cols=%7B%22attributes.weave.client_version%22%3Afalse%2C%22attributes.weave.os_name%22%3Afalse%2C%22attributes.weave.os_release%22%3Afalse%2C%22attributes.weave.os_version%22%3Afalse%2C%22attributes.weave.source%22%3Afalse%2C%22attributes.weave.sys_version%22%3Afalse%7D

Now select the first and last trace - you will get the 2 versions above. However if you select all (with the top checkbox), you only get v1?